kivymd/uix/filemanager.py

```diff
@@ -259,8 +259,7 @@ def events(self, instance, keyboard, keycode, text, modifiers):
 
 
 class BodyManagerWithPreview(MDBoxLayout):
-    """Base class for folder icons and thumbnails images in ``preview`` mode.
-    """
+    """Base class for folder icons and thumbnails images in ``preview`` mode."""
 
 
 class IconButton(CircularRippleBehavior, ButtonBehavior, FitImage):
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a